lib/karafka/logger.rb

```diff
@@ -13,7 +13,7 @@ class Logger < ::Logger
 
     class << self
       # Builds a logger with appropriate settings, log level and environment
```

Comment:
Maybe change docs to `Returns a logger instance with ..` once method does not have `build` name anymore?